app/src/main/java/com/release/keyneez/presentation/main/explore/recent/RecentViewModel.kt

```diff
@@ -0,0 +1,51 @@
+package com.release.keyneez.presentation.main.explore.recent
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.release.keyneez.R
+import com.release.keyneez.domain.model.ExploreData
+
+class RecentViewModel : ViewModel() {
+    private val _itemList = MutableLiveData<List<ExploreData>>()
+    val itemList: LiveData<List<ExploreData>>
+        get() = _itemList
+
+    init {
+        getRecentItemList()
+    }
+
+    private fun getRecentItemList() {
```

Comment:
```suggestion
    private val _activityList = MutableLiveData<List<ExploreData>>()
    val activityList: LiveData<List<ExploreData>>
        get() = _activityList

    init {
        getRecentActivityList()
    }

    private fun getRecentActivityList() {
```
요기도 마찬가지로 직관적인 명칭~
다른 코드에서도 바꿀만한 네이밍이 있는지 한번씩 확인해주세요!

Reply:
넵넵!